crates/runtime/src/http/routes.rs

```diff
@@ -215,8 +217,6 @@ pub(crate) fn routes(
     }
 
     authenticated_router = authenticated_router
-        .layer(Extension(Arc::clone(&rt.app)))
-        .layer(Extension(Arc::clone(&rt.df)))
```

Comment:
Both the app and df references are injected after merging with the unauthenticated router, so there is no need to duplicate them.